### PR TITLE
[JENKINS-46554] alternate for Build with Parameters

### PR DESCRIPTION
**Pull request by mawinter69**
also JENKINS-27214  to override the button on the parameters page

### Testing done

```[tasklist]
### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Customize Build Now Plugin
+==========
+
+This plugin allows users to provide alternate labels to "Build Now", "Build with Parameters" and the "Build" button. 
+While it's well understood by Jenkins experts what "Build Now" means, sometimes its more intuitive for new Jenkins users to associate a 
+more appropriate label to suite the purpose of the job like "Deploy Now" or "Execute Now" or "Promote Now" etc...
```
Comment (timja) [inline README.md:6]:
```suggestion
more appropriate label to suit the purpose of the job like "Deploy Now" or "Execute Now" or "Promote Now" etc...
```

File: src/main/java/org/jenkinsci/plugins/customizebuildnow/AlternateBuildLabelConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package org.jenkinsci.plugins.customizebuildnow;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import jenkins.appearance.AppearanceCategory;
+import jenkins.model.GlobalConfiguration;
+import jenkins.model.GlobalConfigurationCategory;
+
+@org.jenkinsci.Symbol("customBuildNow")
```
Comment (timja) [inline src/main/java/org/jenkinsci/plugins/customizebuildnow/AlternateBuildLabelConfiguration.java:9]: import? the place you copied from must have had a ui symbol in the same class

File: src/main/resources/org/jenkinsci/plugins/customizebuildnow/AlternateBuildLabelConfiguration/help-alternateBuildNow.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<div>
+    Change the default text for the link to build a job without parameters from "Build Now" to, e.g. "Run Now".
+    An individual setting on a job takes precedence over this value.  An empty value means to take the internal Jenkins
```
Comment (timja) [inline src/main/resources/org/jenkinsci/plugins/customizebuildnow/AlternateBuildLabelConfiguration/help-alternateBuildNow.html:3]:
```suggestion
    An individual setting on a job takes precedence over this value. An empty value means to take the internal Jenkins
```

File: src/main/java/org/jenkinsci/plugins/customizebuildnow/AlternateBuildLabelConfiguration.java (hunk 2 above)
Comment (mawinter69) [reply src/main/java/org/jenkinsci/plugins/customizebuildnow/AlternateBuildLabelConfiguration.java]: fixed

**Pull request merged**